### DISCUSSION TITLE
How to manipulate an Image within a Group in Fabric V6

### DISCUSSION THREAD

**Discussion by kiguri** [fabricjs/fabric.js#9320] [General]
I want to perform operations like changing the image source or deleting the image. I have a sandbox here https://codesandbox.io/s/festive-morse-9l3fwk?file=/src/App.js but nothing is working

**Comment by ShaMan123** [comment]
https://codesandbox.io/s/festive-meninsky-82q7wj?file=/src/App.js

search for `// this will work`
